MapboxNavigation/NavigationViewController.swift

```diff
@@ -124,8 +124,10 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
      on the destination of your route. The last coordinate of the route will be
      used if no destination is given.
     */
+    @available(*, deprecated, message: "Destination is no longer support nor necessary. A destination annotation will automatically be added to map given the route.")
```

Comment:
Note that this shouldn’t have much of an impact on most developers: it isn’t possible to customize this annotation (#213) other than changing its title, subtitle, and coordinates, and none of those properties are likely to be modified by the developer anyways.